submissions/luteChongo/html/product-page.html

```diff
@@ -19,7 +19,7 @@ <h1 style="text-align: center; font-family: fantasy;"><u>ZANI STORES</u></h1>
 		<div class="row">
 			<nav>
 				<li><a href="#">Product</a></li>
-				<li><a href="file:///C:/Users/morta/Desktop/Dufuna-CodeCamp-21/submissions/luteChongo/html-forms/contact-form-page.html">Contact Form</a></li>
+				<li><a href="file:///C:/Users/morta/Desktop/Dufuna-CodeCamp-21/Submissions/luteChongo/html-forms/contact-form-page.html">Contact Form</a></li>
```

Comment:
This link is absolute and will only work at your end i.e I don't have this exact directory so it wont work for me. You can simply do '../html-forms/contact-form-page.html'

Also you ought to have the links to both the product page and contact page on both pages.

Reply:
I have done so but it is saying that the file cant be accessed. I have also tried "../html/product-page.html" but it is saying the same thing.
![Screenshot (22)](https://user-images.githubusercontent.com/84315743/120808084-fb568280-c548-11eb-9c67-e0fd05f5a484.png)